webapp/src/Command/ImportEventFeedCommand.php

```diff
@@ -1720,7 +1720,7 @@ protected function importSubmission(array $event)
                 // Submit the solution
                 $contest    = $this->em->getRepository(Contest::class)->find($this->contestId);
                 $submission = $this->submissionService->submitSolution(
-                    $team, $contestProblem, $contest, $language, $filesToSubmit, null,
+                    $team, null, $contestProblem, $contest, $language, $filesToSubmit, null,
```

Comment:
That poses the question whether we need to expose the user to the secondary CCS. It's extremely unlikely that we need to act on this info anyway, but right now the secondary CCS doesn't have enough info, right?

Reply:
Since user is not in the spec at all i wonder how we do this…